components/action/TableauActionsJeune.tsx

```diff
@@ -16,44 +16,44 @@ export const TableauActionsJeune = ({
   actions,
   hideTableHead = false,
 }: TableauActionsJeuneProps) => {
-  const borderStyle = 'border-solid border-0 border-b-2 border-b-grey_700'
-
   return (
-    <div
-      role='table'
-      className='table w-full'
-      aria-label={`Liste des actions de ${jeune.firstName} ${jeune.lastName}`}
-    >
-      <div
-        role='rowgroup'
-        className={`table-header-group ${hideTableHead ? 'sr-only' : ''}`}
-      >
-        <div role='row' className='table-row text-xs-medium text-grey_800'>
+    <>
+      {actions.length === 0 && (
+        <p className='text-md mb-2'>
+          {jeune.firstName} n’a pas encore d’action
+        </p>
+      )}
+
+      {actions.length > 0 && (
+        <div
+          role='table'
+          className='table w-full'
+          aria-label={`Liste des actions de ${jeune.firstName} ${jeune.lastName}`}
+        >
           <div
-            role='columnheader'
-            className={`table-cell pl-4 py-4 ${borderStyle}`}
+            role='rowgroup'
+            className={`table-header-group ${hideTableHead ? 'sr-only' : ''}`}
```

Comment:
```suggestion
            className=`table-header-group`
```
on a plus besoin de la props hideTableHead a priori